IPython/core/oinspect.py

```diff
@@ -827,7 +874,7 @@ def info(self, obj, oname='', formatter=None, info=None, detail_level=0):
             if safe_hasattr(obj, '__call__') and not is_simple_callable(obj):
                 call_def = self._getdef(obj.__call__, oname)
                 if call_def:
-                    call_def = self.format(call_def)
+                    call_def = call_def
```

Comment:
Yeah, I have questions about how format works - which can be done offline. 

In any case, it feels like it should be a formatter and not directly applied in the info dict.